resources/views/layouts/master.blade.php

```diff
@@ -18,6 +17,30 @@
           <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
           <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
         <![endif]-->
+        
+        <!-- EU Cookie Consent Banner -->
+        <link rel="stylesheet" 
+              type="text/css" 
+              href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
```

Comment:
Please don't use cdns, rather `npm install`. Here is the package https://www.npmjs.com/package/cookieconsent.